atlas/atlas.sh

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+source "/usr/local/share/google/dataproc/bdutil/bdutil_helpers.sh"
```

Comment:
We do not source `bdutil` scripts in init actions because we can make braking changes in these scripts in the future and brake init actions. If you need any helper functions, please copy them in the init action.